cli/src/wallet_cli/mod.rs

```diff
@@ -478,6 +480,28 @@ pub async fn claimable_outputs_command(wallet: &Wallet) -> Result<(), Error> {
     Ok(())
 }
 
+// `congestion` command
+pub async fn congestion_command(wallet: &Wallet, account_id: Option<AccountId>) -> Result<(), Error> {
+    let wallet_data = wallet.data().await;
+    let account_id = account_id
+        .or(wallet_data
+            .accounts()
+            .next()
+            .map(|o| o.output.as_account().account_id_non_null(&o.output_id)))
+        .or(wallet_data
+            .implicit_accounts()
+            .next()
+            .map(|o| AccountId::from(&o.output_id)))
+        .ok_or(WalletError::NoAccountToIssueBlock)?;
+    drop(wallet_data);
```

Comment:
Just a tiny bit better I think
```suggestion
    let account_id = {
      let wallet_data = wallet.data().await;
      account_id
          .or_else(|| wallet_data
              .accounts()
              .next()
              .map(|o| o.output.as_account().account_id_non_null(&o.output_id)))
          .or_else(|| wallet_data
              .implicit_accounts()
              .next()
              .map(|o| AccountId::from(&o.output_id)))
          .ok_or(WalletError::NoAccountToIssueBlock)?
    };
```